src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.text.DecimalFormat;
+
+public abstract class Figure implements QuadrangularArea, TriangleArea {
```

Comment:
Don't use verbs for class/interface names. The interface names `QuadrangularArea` and `TriangleArea` suggest actions (calculating area) rather than properties or capabilities of a figure. Consider renaming these interfaces to follow Java naming conventions, such as `Quadrangular` and `Triangular`.